en/docs/install-and-setup/install/installation-options.md

```diff
@@ -0,0 +1,29 @@
+# Installation Options
+
+API Manager and Micro Integrator provide a wide range of installation options for deployments.
+
+1. [Kubernetes(Helm)](#1-kubernetes-helm)
+2. [Docker/Docker Compose](#2-dockerdocker-compose)
+3. [Ansible](#3-ansible)
+4. [Puppet](#4-puppet)
+5. [AWS CloudFormation](#5-aws-cloudformation)
+
+## 1. Kubernetes (Helm)
+
+You can follow the guides available in [https://github.com/wso2/kubernetes-apim](https://github.com/wso2/kubernetes-apim) in order to deploy API Manager and Micro Integrator Kubernetes and Helm artifacts.
```

Comment:
Shouldn't we point to the relevant branch (4.0.x)? Otherwise we might have to update those after the next release

Reply:
Pointed to 4.0.x branches from https://github.com/wso2/docs-apim/pull/4524/commits/485d31bf7416ef451df14101edae3c0c1bdba094